wiki/Beatmap/Difficulty/id.md

```diff
@@ -57,17 +66,17 @@ Berikut adalah nama tingkat kesulitan yang biasa dimiliki dalam beatmap. Definis
 - ![](/wiki/shared/diff/expert-m.png) Expert
 - ![](/wiki/shared/diff/expertplus-m.png) Expert+
 
-## Star rating (tingkat kesulitan dalam skala bintang)
+## Star rating
 
 ![Jangkauan star rating](img/SR-range.png)
 
-Di situs web, beatmap secara otomatis diberikan ikon kesulitan berdasarkan rentang *star rating* ini:
+Situs web osu! secara otomatis mengelompokkan beatmap-beatmap yang diunggah ke dalam enam rentang tingkat kesulitan berdasarkan star rating dari beatmap yang bersangkutan sebagai berikut:
```

Comment:
```suggestion
Situs web osu! secara otomatis mengelompokkan beatmap-beatmap yang diunggah ke dalam enam rentang tingkat kesulitan berdasarkan star rating dari beatmap yang bersangkutan:
```
colon already represesnts `, yaitu`

Reply:
think i'll keep this one as `sebagai berikut:` is the indonesian equivalent of `as follows:`